packages/suricata.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,114 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+Import-Module powershell-yaml
+
+# set configurations
+$toolName = "Suricata"
+$category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+$filetype = "MSI"
```

Comment:
Use the `MSI` directly, since it is a short string and it is only used once. This is consistent with other packages.
```suggestion
```